aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -22,39 +18,69 @@ We exercise empathy in our voice through situational tones that speak to each co
 - Fact-based
 - Plain English
 
+Refer to the guidance for [HPE's Voice](https://brandcentral.hpe.com/brand-central/content/voice) for more information.
+
 ## Best Practices
 
 The following are some high-level guidelines for constructing good messaging.
 
-## Guidelines for Constructing Good Messaging
-
-- **Use HPE's voice and tone**
-  - Conversational American English
-  - Correct grammar and punctuation
-  - Active tense
-  - Human and approachable, not robotic sounding
-  - Positive, "yes we can," not negative
-  - Never condescending
-- **Write for all reading levels**
-  - To ensure clear, straightforward and efficient communication, a good benchmark to consider while writing is a United States grade 7 reading level. Check for grade reading level using an app like [Hemingway App](http://www.hemingwayapp.com/).
-- **Respect the user and their time**
-  - Establish context
-  - Lead with what is most important
-  - Explain using terminology consistent with the UI
-  - Point the user directly to a resolution. Do not use "_contact your
-    administrator_" or "_contact HPE_" &mdash; the administrator is likely
-    the user and the support contact could be a channel partner. Instead,
-    provide a direct means to connect the user with the party able to resolve
-    the issue.
-- **Inspire trust and confidence**
-  - Use informative, concise language
-  - State facts without coloring them in opinions
-  - Avoid charged words such as "_error_"
-  - Focus on the solution, avoid casting blame on the user or software
-  - Focus on what is needed to be accomplished, avoid personal pronouns
-    (e.g. I, you, he, she, it, we, they, me, him, her, us, them)
-- **Ensure the voice is human and humane**
-  - Read your message out loud to another person. Is this how you would say it out loud? Does the other person feel this is how two people would converse?
+## Writing Effective Content
+
+A design speaks for itself. 
+
+It communicates the interaction of the user experience; how a task is accomplished.
+
+Content supplements design.
+
+It communicates the goal of the user experience; why a task should be accomplished.
+
+We can create meaningful user experiences by writing clarity into our content; what we say and how we say it. Clarity simplifies complex ideas and connects one idea to the next to bridge the gap between user and application, task and goal. 
+
+We can write clear content with a few simple qualities: 
+
+### Accessible
+
+Write with clear language to communicate clear ideas.
+
+Write to users in plain language, grounded in positive ability and action. Plain language requires less cognitive effort from users, improving time to action. Write at a United States grade 7 reading level to create straightforward, succinct, and consistent content.
```

Comment:
It'd be great to provide a few examples of "plain language". Perhaps in a Do-Don't format.

What are common violations?

utilize --> use
excessive adverbs --> so very interesting --> fascinating
opening a window that contained blah blah inside  --> opening a window containing blah blah



Not neccessary for right now. Just a thought for the future.

Reply:
This is all good feedback! I'm taking notes for future updates.